src/Sulu/Bundle/RouteBundle/Controller/RouteController.php

```diff
@@ -109,7 +109,7 @@ public function cgetAction(Request $request)
         // optional parameter
         $history = $this->getBooleanRequestParameter($request, 'history', false, false);
         $mapping = $this->resourceKeyMappings[$resourceKey] ?? [];
-        $entityClass = $mapping['entityClass'] ?? null;
+        $entityClass = $this->getRequestParameter($request, 'entityClass') ?? $mapping['entityClass'] ?? null;
```

Comment:
TODO: add a testcase for the changes in this controller

Reply:
Can you add this missing test?

Reply:
done 😇